client/node_client.go

```diff
@@ -58,37 +58,31 @@ type NodeWebsocketClient interface {
 	Close()
 }
 
-// NOTE [ben] Compiler check to ensure ErisNodeClient successfully implements
+// NOTE [ben] Compiler check to ensure erisNodeClient successfully implements
 // eris-db/client.NodeClient
-var _ NodeClient = (*ErisNodeClient)(nil)
+var _ NodeClient = (*erisNodeClient)(nil)
 
 // Eris-Client is a simple struct exposing the client rpc methods
-
-type ErisNodeClient struct {
+type erisNodeClient struct {
```

Comment:
This was a hygiene thing. Note that the methods are on the interface `NodeClient`. Generally I avoid exporting the concrete struct type that implements and interfaces because I don't want people constructing it without using my constructor, `NewErisNodeClient` in this case. Bear in mind you don't need to export the struct to use the type via the exported constructor. I use this pattern a lot: https://github.com/eris-ltd/eris-db/blob/feature_logging/logging/loggers/info_trace_logger.go#L16.

In this case I particularly thing it should be unexported because it has private fields so an external package won't be able to initialise it properly for certain.